src/traccuracy/utils.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import annotations
+
+import copy
+from typing import TYPE_CHECKING
+
+import numpy as np
+
+from traccuracy._tracking_graph import NodeFlag, TrackingGraph
+
+if TYPE_CHECKING:
+    from traccuracy.matchers._base import Matched
+
+
+def get_corrected_division_graphs_with_delta(
+    matched: Matched, frame_buffer: int = 0
+) -> tuple[TrackingGraph, TrackingGraph]:
+    """Returns copies of graphs with divisions corrected.
+
+    All divisions corrected by a frame_buffer value less than or equal
+    to the given frame buffer are marked as `TP_DIV`.
+
+    Args:
+        matched (traccuracy.matchers.Matched): Matched object for set of GT and Pred data.
+            Must be annotated with division events.
+        frame_buffer (int): Maximum frame buffer to use for division correction
+
+    Returns:
+        tuple[traccuracy.TrackingGraph, traccuracy.TrackingGraph]: Tuple of corrected
+        GT and Pred graphs
+    """
+    corrected_gt_graph = copy.deepcopy(matched.gt_graph)
```

Comment:
We can check the `matched.gt_graph.division_annotations` flag to check that the divisions have been annotated.